cropharvest/eo/eo.py

```diff
@@ -158,19 +180,33 @@ def _filter_labels(self, labels: geopandas.GeoDataFrame) -> geopandas.GeoDataFra
 
     @staticmethod
     def _export(
-        image: ee.Image, region: ee.Geometry, filename: str, description: str, drive_folder: str
+        image: ee.Image,
+        region: ee.Geometry,
+        filename: str,
+        description: str,
+        drive_folder: Optional[str] = None,
+        dest_bucket: Optional[str] = None,
+        file_dimensions: Optional[int] = None,
     ) -> ee.batch.Export:
 
-        task = ee.batch.Export.image.toDrive(
+        kwargs = dict(
             image=image.clip(region),
-            fileNamePrefix=filename,
             description=description[:100],
```

Comment:
Should we have a check that the description is not longer than 100 characters?

Reply:
I guess we could but I don't know if that would be all that helpful. Do you think otherwise? 
Just forcefully concatenating makes it so that the user of this function does not have to be aware of this GEE limitation.

Reply:
That makes sense. I think it could just lead to unexpected behavior when the description in Earth Engine is not the same as what was passed here (if it is longer than 100 characters). 

However since `description` is not publicly exposed this is probably fine to leave as is.